specifications/08-variants.json

```
@@ -205,6 +205,27 @@
                         }
                     }
                 ]
+            },
+            {
+                "name": "Feature.Variants.H",
+                "description": "variant with payload number",
```

I think this signals the intent better

```suggestion
                "description": "variant with numeric payload",
```